test/support/utils_sycl.h

```diff
@@ -157,13 +157,27 @@ struct invoke_on_all_hetero_policies
     }
 };
 
+bool has_aspect(const sycl::device& device, sycl::aspect aspect)
+{
+    try
+    {
+        return device.has(aspect);
+    }
+    catch(sycl::exception&)
```

Comment:
sycl::exception& or const sycl::exception& ?
And may be better to log exception text into std::cerr or std:: cout ?

Reply:
We do not modify the exception object, that is why using ```const``` is right. 
I added logging to ```count``` for notification purpose. ```count``` is used instead of ```cerr```, because non-implemented aspects are specific to the environment we support, it should not be treated as an error.